pkg/api/handlers/libpod/containers.go

```diff
@@ -402,18 +402,46 @@ func InitContainer(w http.ResponseWriter, r *http.Request) {
 func UpdateContainer(w http.ResponseWriter, r *http.Request) {
 	name := utils.GetName(r)
 	runtime := r.Context().Value(api.RuntimeKey).(*libpod.Runtime)
+	decoder := utils.GetDecoder(r)
+	query := struct {
+		RestartPolicy  string `schema:"restartPolicy"`
+		RestartRetries uint   `schema:"restartRetries"`
+	}{
```

Comment:
you need to update the swagger docs to include the new params for the libpod endpoint

Reply:
Already done

Reply:
I don't see that, you need to add the params to the libpod endpoint but you added the docs for the compat endpoint

Reply:
Damn it. Fixed.